src/TraceMiddleware.php

```diff
@@ -285,7 +285,7 @@ private function createHttpDebug(CommandInterface $command)
 
     private function flushHttpDebug(CommandInterface $command)
     {
-        if ($res = $command['@http']['debug']) {
+        if ($this->config['http'] && $res = $command['@http']['debug']) {
```

Comment:
In case the underlying logic changes later, wouldn't it be safer to change it to if (isset($command['@http']['debug']) && $res = $command['@http']['debug'])?